src/backend/metal/src/device.rs

```diff
@@ -150,11 +150,8 @@ impl Drop for Device {
     fn drop(&mut self) {
         if cfg!(feature = "auto-capture") {
```

Comment:
let's remove the `auto-capture` feature entirely now, please.
If the user wants it, they can control it now